teku/src/main/java/tech/pegasys/teku/cli/subcommand/MigrateDatabaseCommand.java

```diff
@@ -103,22 +127,29 @@ public void run() {
     // validate there is no old database instance present
     // If a previous migrate-data was run, the 'beacon' would have been renamed to 'beacon.old'
     // and advice given to the user to cleanup 'beacon.old' manually.
-    if (Files.isDirectory(dataOptions.getDataBasePath().resolve("beacon.old"))) {
+    if (Files.isDirectory(dataPath.resolve("beacon.old"))) {
       SUB_COMMAND_LOG.exit(
           1,
           "There is an existing folder in "
-              + dataOptions.getDataBasePath().resolve("beacon.old").toFile()
+              + dataPath.resolve("beacon.old").toFile()
               + ", review this folder and remove before continuing.");
     }
+
+    DataConfig.Builder dataConfigBuilder = DataConfig.builder().dataBasePath(dataPath);
+    if (dataBeaconPath != null) {
+      dataConfigBuilder.beaconDataPath(dataBeaconPath);
+    }
```

Comment:
nit: I wonder if the builder should just handle the null check to simplify calling code.